File: check-if-n-and-its-double-exist.py
class Solution(object):
    def checkIfExist(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        a=len(arr)
        for i in range(0,a):
            for j in range(i+1,a):
                if (arr[i]==2*arr[j]) or (arr[j]==2*arr[i]):
                    return True

File: check-if-n-and-its-double-exist.java
class Solution {
    public boolean checkIfExist(int[] arr) {
        HashSet<Integer> set=new HashSet<>();
        for(int i:arr){
            if(set.contains(2*i) || (set.contains(i/2) && i%2==0)){
                return true;
            }     
            set.add(i);
        }
        return false;
    }
}
